managed-identity-storage-blob/README.md

```diff
@@ -0,0 +1,51 @@
+# Access Storage Blob with Managed Identity
+
+This sample shows how to access Storage Blob with Managed Identity in `Azure Spring Cloud`.
+
+## Prerequisite
+
+* [JDK 8](https://docs.microsoft.com/en-us/azure/java/jdk/java-jdk-install)
+* [Maven 3.0 and above](http://maven.apache.org/install.html)
+* [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) or [Azure Cloud Shell](https://docs.microsoft.com/en-us/azure/cloud-shell/overview)
+
+## How to run 
+
+1. Run `mvn clean package`.
+2. Install Azure CLI extension for Azure Spring Cloud by running below command.
+    ```
+    az extension add -y --source https://azureclitemp.blob.core.windows.net/spring-cloud/spring_cloud-0.1.0-py2.py3-none-any.whl
+    ```
+3. Create an instance of Azure Spring Cloud.
+    ```
+    az spring-cloud create -n <resource name> -g <resource group name>
+    ```
+4. Create an app with public domain assigned.
+    ```
+    az spring-cloud app create -n <app name> -s <resource name> -g <resource group name> --is-public true 
+    ```
+5. Enable system assigned Managed Identity.
```

Comment:
Managed identity doesn't have to be capitalized.
-> `Enable system-assigned managed identity for your app and take note of the principal id from the command output.`